fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ExportCommand.java

```diff
@@ -49,35 +100,248 @@ public class ExportCommand extends Command implements ForwardWithSync {
     public ExportCommand(List<String> nameParts, List<String> partitions, String whereSql, String path,
             Map<String, String> fileProperties, BrokerDesc brokerDesc) {
         super(PlanType.EXPORT_COMMAND);
-        this.nameParts = nameParts;
+        this.nameParts = ImmutableList.copyOf(Objects.requireNonNull(nameParts, "nameParts should not be null"));
+        this.path = Objects.requireNonNull(path.trim(), "export path should not be null");
         this.partitionsNameList = partitions;
         this.whereSql = whereSql;
-        this.path = path.trim();
         this.fileProperties = fileProperties;
-        this.brokerDesc = brokerDesc;
+        if (brokerDesc == null) {
```

Comment:
should not pass `null`. u could use `Optional`

Reply:
done